/opt/local/var/macports/sources/rsync.macports.org/release/ports/devel/cgdb/work/cgdb-0.6.4 /opt/local/var/macports/sources/rsync.macports.org/release/ports/devel/cgdb
--- various/util/src/pseudo.c.orig	2009-05-24 14:59:19.000000000 -0700
+++ various/util/src/pseudo.c	2009-05-24 14:59:55.000000000 -0700
@@ -331,6 +331,7 @@
 
 #ifndef HAVE_CYGWIN 
 #ifndef HAVE_LINUX /* linux does not use the streams module */
+#ifndef __APPLE__
 	/*
 	 * Push the appropriate streams modules, as described in Solaris pts(7).
 	 * HP-UX pts(7) doesn't have ttcompat module.
@@ -359,6 +360,7 @@
 #endif
 #endif
 #endif
+#endif
 
 #else /* HAVE_DEV_PTMX */
 #ifdef HAVE_DEV_PTS_AND_PTC
